src/transformers/adapters/loading.py

```diff
@@ -52,6 +55,8 @@ def save_weights_config(self, save_directory, config, meta_dict=None):
             for k, v in meta_dict.items():
                 if k not in config:
                     config[k] = v
+        # add version to config
+        config["adapter_version"] = __adapters_version__
```

Comment:
Would it make sense to move this to the `build_full_config()` method in configuration.py as similar data is added over there?